### PR TITLE
Blog post with organizational audit

### PR DESCRIPTION
**Pull request by choldgraf**
This post describes the work that Davina has recently done with 2i2c, and links out to a Zenodo entry (yet to be posted) with the full organizational report.

```[tasklist]
- [x] Finalize language for this post and fix links
- [x] Post to Zenodo
- [ ] Blog post goes live
```

### CODE REVIEW

**Comment by choldgraf** [comment]
Got a +1 from Davina on the language of this + confirmation that no changes are needed to her report.

File: content/blog/2023/organizational-report/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Open organizational report: Strengths and challenges for 2i2c's team"
+subtitle: ""
+summary: ""
+authors: ["Chris Holdgraf"]
+tags: []
+categories: ["updates"]
+date: 2023-11-03
```
Comment (damianavila) [inline content/blog/2023/organizational-report/index.md]: Should be updated before going live.

**Review by damianavila — APPROVED** [review]
Text LGTM (except for a minor fix on the date), @choldgraf.

File: content/blog/2023/organizational-report/index.md (hunk 1 above)
Comment (choldgraf) [inline content/blog/2023/organizational-report/index.md]:
```suggestion
date: 2023-11-08
```

**Pull request merged**